src/common/types.cpp

```diff
@@ -153,6 +153,9 @@ const vector<LogicalType> LogicalType::INTEGRAL = {LogicalType::TINYINT,   Logic
                                                    LogicalType::BIGINT,    LogicalType::HUGEINT,  LogicalType::UTINYINT,
                                                    LogicalType::USMALLINT, LogicalType::UINTEGER, LogicalType::UBIGINT};
 
+const vector<LogicalType> LogicalType::TEMPORAL = {LogicalType::DATE, LogicalType::TIMESTAMP, LogicalType::TIME,
```

Comment:
Is this used anywhere?

Reply:
Not any more. I was using it for the loop before I killed the unsigned numerics. I can remove it if you like, but its one of those things that if you don't have it people might duplicate it all over the place.